02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>why would people need to do this?</p><p>can you give some context to when this would be a beneficial use of time?</p><p>Every decision is a cost-benefit analysis, so before I endeavor to changes something (cost) I need to know what I will be able to do in the future that I can't do today (benefit)</p><p>Add some context into this section (or delete it?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I added the following explanation:</p><p> </p><p>"using the&nbsp;<code>name</code>&nbsp;property may be more helpful when trying to execute a&nbsp;<code>for</code>&nbsp;loop to iterate over all hierarchies in an Atoti data model, where we may want the colloquial hierarchy name rather than the fully qualified one (with both the dimension and hierarchy listed)."</p><p> </p><p>But I inferred this based on what I see, i.e. something like <code>h['Address'].name</code> seems super redundant to me so I can only imagine that the property comes to play when iterating over hierarchies similar to what I explained. If there is a more specific use case or description for <em>why</em> we need this property, I think someone like @tibdex or another R&amp;D person would have to chime in. </p>